src/frontend/screens/Settings/CreateTestData.tsx

```diff
@@ -258,7 +258,7 @@ function useCreateFakeObservationsMutation() {
       const promises = [];
 
       for (let i = 0; i < count; i++) {
-        const fakeCoordinates = randomPosition({
+        const [lon = 0, lat = 0] = randomPosition({
```

Comment:
nit: would probably prefer to just throw if `randomPosition()` doesn't return a tuple of 2 elements here. 0 is unfortunately a [significant value](https://en.wikipedia.org/wiki/Null_Island) in the context of geospatial coordinates.

the TS typing they have (`number[]`) is inaccurate, as the implementation always returns a tuple:

https://github.com/Turfjs/turf/blob/6889237200a088fa5539f92f361fc96135e09ac1/packages/turf-random/index.ts#L30-L44

Reply:
Fixed in 9142abf01fbfad057de4c22a46e03fc49b4cb586. Also opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70521 to fix the upstream types.